packages/devtools/lib/src/inspector/inspector.dart

```diff
@@ -187,15 +190,28 @@ class InspectorScreen extends Screen {
 
         displayedWidgetTrackingNotice = true;
 
-        framework.showWarning(
-            message: '''The widget creation tracking feature is not enabled;
-this feature allows the Flutter inspector to present the widget tree in a manner
-similar to how the UI was defined in your source code. Without it, the tree of
-nodes in the widget tree are much deeper, and it can be more difficult to
-understand how the runtime widget hierarchy corresponds to your application’s UI.
-
-To fix this, relaunch your application by running 'flutter run
---track-widget-creation' (or run your application from VS Code or IntelliJ).''');
+        framework.showWarning(children: <CoreElement>[
+          div()
+            ..add(span(text: 'The '))
+            ..add(a(
+                text: 'widget creation tracking feature',
+                href: trackWidgetCreationDocsUrl,
+                target: '_blank;'))
+            ..add(span(text: ' is not enabled. '))
+            ..add(span(
+                text:
+                    '''This feature allows the Flutter inspector to present the 
+                    widget tree in a manner similar to how the UI was defined in
```

Comment:
nit: this will add a bunch of leading spaces in the HTML not that it really matters.
Consider the strange indent from the previous version where subsequent lines are aligned all the way on the left.